### ISSUE TITLE
Dialog is break

### ISSUE THREAD

**Issue by vincehi** [cschroeter/park-ui#467]
<img width="1087" alt="Capture d’écran 2024-11-07 à 09 23 04" src="https://github.com/user-attachments/assets/7b70e042-f3a1-4394-aef3-038f304aee15">

**Comment by riceboyler** [comment]
Yeah, and I personally can't even get it to load the backdrop for me when I try to run it in my local project. I've done the CLI installation, but `dialog__backdrop` doesn't appear to be getting generated by panda, even after running prepare. Something feels just plain off about the implementation of Dialog.

**Comment by thewebartisan7** [comment]
It seems an issue just with demo.
The buttons should not have `width=full` here in demo https://github.com/cschroeter/park-ui/blob/main/components/react/src/demos/dialog.demo.tsx#L21C1-L28C21 
by removing this works fine.
in case it should have `flex: 1` to make both buttons full width inside dialog.
